### PR TITLE
path, links, and screenshots

### PR DESCRIPTION
**Pull request by WenxinM**
fix the github path, ad the syzygy link
@wupeifa I assume that the first syzygy link should be `pims.syzygy.ca` rather than the `quantecon.syzygy.ca`. I corrected it here but it was redirected to the quantecon one in the source repo.

### CODE REVIEW

File: src/introduction/local_install.md [hunk 1]
```diff
@@ -56,17 +56,17 @@ You can download the lectures through either **Github Desktop** or **Terminal**:
 1. Install [Github Desktop](https://desktop.github.com/).
 1. Click the "Open in Github Desktop" option in the clone button menu. It should open a Github
    Desktop popup that looks like this:
-   ```{figure} https://datascience.quantecon.org/assets/_static/introduction_files/download_lectures_github_desktop.png
+   ```{figure} ../_static/download_lectures_github_desktop.png
    ```
    You should choose the path (folder) where you would like to download the repository. The default path on
-   Windows should be `C:/Users/YOUR_USERNAME/Documents/GitHub`.
+   Windows should be `C:\Users\YOUR_USERNAME\Documents\GitHub`.
```
Comment (wupeifan) [inline src/introduction/local_install.md:62]: This would cause some issues when copying onto Windows, I think. Either `/` or `\\` works. Just my 2cents.

Comment (aadsouza) [reply src/introduction/local_install.md:62]: @wupeifan whoops, I had thought this was a typo. i recall that windows usually used `\` in their paths unlike MacOS etc. when one copies and pastes into the terminal, does windows flip them around? can't really test on my  devices. will change it back if you think i should. (@WenxinM any thoughts?)

also found some discussion here https://stackoverflow.com/questions/1589930/so-what-is-the-right-direction-of-the-paths-slash-or-under-windows

Comment (aadsouza) [reply src/introduction/local_install.md:62]: okay, will revert this change.

**Comment by wupeifan** [comment]
If there's no additional issue related to this one then I think this might be good to merge as well?

**Comment by aadsouza** [comment]
@wupeifan we were waiting for the theme to be merged in and to finish up the rest of the screenshots

**Comment by wupeifan** [comment]
Ah gotcha, I don't want to interrupt your workflow :) just ignore my comments

**Comment by aadsouza** [comment]
> @arnavs except for theme-specific screenshots for which we can create a new PR, this takes care of some of the path stuff - i think this is good to go; hope it doesn't conflict with what you had intended to do with the revised setup instruction stuff

@wupeifan oops hahah, i forgot that i had said this. @WenxinM is this good to merge?

**Comment by aadsouza** [comment]
what do we think about the section specific icons being used for each subsection?

**Pull request merged**